src/moar.h

```diff
@@ -63,6 +63,8 @@ typedef double   MVMnum64;
 #define ALIGNOF(t) ((char *)(&((struct { char c; t _h; } *)0)->_h) - (char *)0)
 #endif
 
+#define ASSERT_ALIGNED(var, align) assert(!((MVMuint64)(var) % (MVMuint64)(align)))
```

Comment:
Is it justified to pollute `src/moar.h` with this macro?

Reply:
Also, running on `%` on non-constants is could be costly computationally.

Reply:
I'd prefer that any macros we add have the `MVM_` prefix, so we know they are ours and so we don't pollute things for any embedders.